docs/admin/Offline_Cert_Renewal.md

```diff
@@ -0,0 +1,178 @@
+Offline Cert Renewal
```

Comment:
I think we should call it "Offline System Certificate Renewal" to distinguish it from regular cert renewal. Let's change the filename too.

Also, try to use the full words (e.g. "certificate" instead of "cert", "temporary" instead of "temp") in the doc text. It's OK to use the shortened words in the commands.